web/locale/en_US/messages.po

```diff
@@ -3292,3 +3292,41 @@ msgstr "This is optional, but will allow your child to log in by themselves to a
 
 msgid "You must provide an email address when providing a password."
 msgstr "You must provide an email address when providing a password."
+
+msgid "Type the content of your email here"
+msgstr "Type the content of your email here"
+
+msgid "Email sent!"
+msgstr "Email sent!"
+
+msgid "An error has occurred while sending the email"
+msgstr "An error has occurred while sending the email"
+
+msgid "Send Email"
+msgstr "Send Email"
+
+msgid "From"
+msgstr "From"
+
+msgid "To"
+msgstr "To"
+
+msgid "Included Parents"
+msgstr "Included Parents"
+
+msgid "Email subject"
+msgstr "Email subject"
+
+msgid "Email subject must not be empty"
+msgstr "Email subject must not be empty"
+
+msgid "Email content is required"
```

Comment:
I know this existed in previous PRs, but... Consistency? Email subject must not be empty, but email content is required 😛